### ISSUE TITLE
'N/A' text when thermostat is off

### ISSUE THREAD

**Issue by terafin** [nervetattoo/simple-thermostat#51]
Hi there! Thanks again for the great custom card - but sadly it looks like 'null' still appears in the latest release (0.23) when the thermostat is off.

**Comment by nervetattoo** [comment]
Exactly where does null appear? Screenshot is always appreciated :)

**Comment by terafin** [comment]
I'm so sorry! Worst bug report ever :)
<img width="421" alt="Screen Shot 2019-05-24 at 2 41 30 PM" src="https://user-images.githubusercontent.com/66893/58358043-10648b00-7e32-11e9-9934-c2627951ddac.png">

**Comment by terafin** [comment]
Updated from null to N/A

**Comment by nervetattoo** [comment]
This is pretty much as expected. I need to think about how to handle device integrations that stop reporting target temp when they are turned off. I guess at least the up/down buttons should both be disabled, but isn't Not Available a good enough value in this scenario?

**Comment by terafin** [comment]
It would be great to say "Off" or something. "N/A" seems very nerdy/geeky for the guests of my house at least :) (Which is what I use the dashboard for!)

**Comment by nervetattoo** [comment]
Right. Can you export the full state attributes for the device when it is off? I'll see how it fits

**Comment by terafin** [comment]
Thanks! Here's a couple screenshots (one on, one off)

<img width="818" alt="Screen Shot 2019-06-06 at 10 44 17 AM" src="https://user-images.githubusercontent.com/66893/59054235-143bd880-8848-11e9-8bd9-6b21a0967eca.png">
<img width="820" alt="Screen Shot 2019-06-06 at 10 44 14 AM" src="https://user-images.githubusercontent.com/66893/59054236-14d46f00-8848-11e9-98c9-bb01d5e220fe.png">

**Comment by asev** [comment]
Hello. Thanks @nervetattoo for this elegant thermostat card. I'm using it for my heater which is active only in winter time. As it's already April I switched it off and noticed those N/A values.
![image](https://user-images.githubusercontent.com/10020928/78181117-854e1b80-746c-11ea-994c-74362ed36670.png)

I would expect to see current temperature instead of N/A. Also `step_layout` option may accept `false` for removal.

In other words it would be nice to be able to configure the card to look like this. (_22.5℃ is current temperature_)
![image](https://user-images.githubusercontent.com/10020928/78181409-00afcd00-746d-11ea-9200-72ef965d7533.png)

Let me know what you think. :)
